### ISSUE TITLE
Massive console spam

### ISSUE THREAD

**Issue by MechanosG** [ExcessiveAmountsOfZombies/Croptopia#46]
**Version**
1.2.0 

**Platform**
Fabric

**Describe the issue**
While grabbing my log to debug a different issue for another mod, to my surprise my log is 376,000+ lines long, on a fresh game start!  This is line 618 
```
[19:00:58] [Worker-Main-2/WARN]: Unable to resolve texture due to reference chain crop->crop in croptopia:block/croptopia_leaves
```
and it repeats 'till line 375017
```
[19:01:22] [Worker-Main-2/WARN]: Unable to resolve texture due to reference chain crop->crop in croptopia:block/croptopia_leaves
```
So it seems to be happening all at once, in under 30 seconds, then stops.  The log bloated out to 48MB.

**To Reproduce**
Launch a modpack with Croptopia singleplayer, look at console?

**Screenshots**
N/A

**Other Mods**
This modpack currently has like...  180+ mods or something.  So I'll just copy the list from a recent unrelated crashlog that should be mostly accurate.
```
Fabric Mods: advanced_runtime_resource_pack: Runtime Resource Pack 0.2.9
               adventurez: AdventureZ 1.1.5
               allure: Allure 1.2.8
               amecs: Amecs 1.3.2+mc.1.16.3
               amecsapi: Amecs API 1.1.1+mc1.16-rc1
               ancientgateways: Ancient Gateways 1.0.2
               anonymize: Anonymize 0.1.0
               aopalliance_aopalliance: aopalliance 1.0
               appleskin: AppleSkin 1.0.11
               appliedenergistics2: Applied Energistics 2 8.1.0-alpha.13
               artifice: Artifice 0.13.2+1.16.3
               astromine: Astromine 1.11.5+fabric-1.16.5
               astromine-core: Astromine: Core 1.11.5+fabric-1.16.5
               astromine-discoveries: Astromine: Discoveries 1.11.5+fabric-1.16.5
               astromine-foundations: Astromine: Foundations 1.11.5+fabric-1.16.5
               astromine-technologies: Astromine: Technologies 1.11.5+fabric-1.16.5
               astromine-transportations: Astromine: Transportations 1.11.5+fabric-1.16.5
               authme: Auth Me 1.4.0
               autoconfig1u: Auto Config v1 Updated 3.3.1
               autofish: Autofish 0.9.2
               automated_crafting: Automated Crafting 1.3.14+MC1.16.2-1.16.4
               automotons: Automotons 1.0.1
               balance_of_exile: Balance of Exile 1.0.3
               bedrockify: BedrockIfy 0.2.0
               bettercaves: YUNG's Better Caves 1.16.2-1.0.3
               betterdroppeditems: Better Dropped Items 1.2.4-1.16.2
               bettereating: Better Eating 0.2.0
               betterend: Better End 0.8.0-beta
               bettermineshafts: YUNG's Better Mineshafts 1.16.4-1.0
               betternether: Better Nether 5.0.4
               betterstones: Better Stones 0.0.4
               biomemakeover: Biome Makeover 1.16.4-1.1.5
               blade: Blade 2.0.0
               blame: Blame 2.3.0
               blast: BLAST 1.5
               blockus: Blockus 2.0.1+1.16.4
               blue_endless_jankson: jankson 1.2.0
               breakprogress: Break Progress 1.0.1
               buildersblocks: Builders' Blocks 1.16.3-beta.8
               cameraoverhaul: Camera Overhaul 1.2.2-fabric-universal
               campanion: Campanion 1.3.1+build.51
               can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.0.2
               canvas: Canvas Renderer 1.0.1257
               cardinal-components: Cardinal Components API 2.7.9
               cardinal-components-base: Cardinal Components API (base) 2.7.10
               cardinal-components-block: Cardinal Components API (blocks) 2.7.10
               cardinal-components-chunk: Cardinal Components API (chunks) 2.7.10
               cardinal-components-entity: Cardinal Components API (entities) 2.7.10
               cardinal-components-item: Cardinal Components API (items) 2.7.10
               cardinal-components-level: Cardinal Components API (world saves) 2.7.10
               cardinal-components-scoreboard: Cardinal Components API (scoreboard) 2.7.9
               cardinal-components-util: Cardinal Components API (utilities) 2.7.10
               cardinal-components-world: Cardinal Components API (worlds) 2.7.10
               carpet: Carpet Mod in Fabric 1.4.24
               chainmail: Chainmail 0.5.0
               chat_heads: Chat Heads 0.2.1+1.16.2
               cinderscapes: Cinderscapes 1.3.0+build.51
               cleancut: CleanCut mc1.16-2.1-fabric
               clearview: Clear View 2.1
               clickthrough: ClickThrough 1.16.4-fabric0.25.1-0.3
               climbable: Climbable 0.2.0
               cloth-armor-api-v1: Cloth Armor API v1 1.4.8
               cloth-basic-math: Cloth Basic Math 0.5.1
               cloth-client-events-v0: Cloth Client Events v0 1.4.5
               cloth-common-events-v1: Cloth Common Events v1 1.4.8
               cloth-config2: Cloth Config v4 4.8.3
               cloth-datagen-api-v1: Cloth Datagen v1 1.4.9
               cloth-durability-bar-api-v1: Cloth Durability Bar API v1 1.4.9
               cloth-dynamic-registry-api-v1: Cloth Dynamic Registry API v1 1.2.1
               cloth-utils-v1: Cloth Utils v1 1.4.5
               coloredcraftingstations: ColoredCraftingStations 1.0.1
               colormatic: Colormatic 2.2.7+mc.1.16.3
               com_electronwill_night-config_core: core 3.6.0
               com_electronwill_night-config_toml: toml 3.6.0
               com_fasterxml_jackson_core_jackson-annotations: jackson-annotations 2.9.0
               com_fasterxml_jackson_core_jackson-core: jackson-core 2.9.0
               com_fasterxml_jackson_core_jackson-databind: jackson-databind 2.9.0
               com_fasterxml_jackson_dataformat_jackson-dataformat-yaml: jackson-dataformat-yaml 2.9.0
               com_fasterxml_jackson_module_jackson-module-kotlin: jackson-module-kotlin 2.9.0
               com_github_shevek_parallelgzip: parallelgzip master-SNAPSHOT
               com_github_therandomlabs_trlutils-config: TRLUtils-Config master-SNAPSHOT
               com_github_therandomlabs_trlutils-fabric: TRLUtils-Fabric 1.16-SNAPSHOT
               com_google_inject_guice: guice 4.2.3
               com_moandjiezana_toml_toml4j: toml4j 0.7.2
               com_typesafe_config: config 1.4.0
               com_udojava_evalex: EvalEx 2.5
               computercraft: CC:T for Fabric 1.91.2
               connected_block_textures: Connected Block Textures 0.1.2+1.16.4
               convenient-mobgriefing: Convenient mobGriefing 1.1.4
               cotton-config: Cotton Config 1.0.0-rc.7
               cotton-logging: Cotton Logging 1.0.0-rc.4
               couplings: Couplings 1.3.3
               cpas: conveniently placed armor slots 1.2.0+1.16.4
               creeperspores: Creeper Spores 1.4.1
               croptopia: Croptopia 1.0.5-1.16.3-FABRIC
               crowdin-translate: CrowdinTranslate 1.2
               crowmap: Crowmap 1.0.1
               ctm: ConnectedTexturesMod for Fabric 0.1.0
               cullparticles: Cull Particles 2.0
               curios: Curios API 0.0.10-1.16.3
               customspawns: Custom Spawns 0.1.2
               customsplashscreen: Custom Splash Screen 1.0.0
               dankstorage: Dank Storage 1.9a-1.16.3
               darkness: True Darkness 1.6.57
               dawn: Dawn API 1.6
               demeter: Demeter 2.0.0
               disable_custom_worlds_advice: Disable Custom Worlds Advice 1.2
               dontdropit: Don't Drop It! 2.2.0+1.16
               doom: Doom Fabric 3.9.3
               dynamicfps: Dynamic FPS 2.0.1
               earthtojavamobs: Earth2Java 1.5.0+1.16.4
               easiercrafting: EasierCrafting 1.16.4-fabric0.25.1-1.6.6
               edit_sign: Edit Sign 2.1.3
               enchant_giver: Enchant Giver 1.0.2
               expandedstorage: ExpandedStorage 5.6.31
               ezpas: Ezpas 1.2.3-1.16.2
               fabric: Fabric API 0.28.3+1.16
               fabric-api-base: Fabric API Base 0.2.0+ab87788d3a
               fabric-biome-api-v1: Fabric Biome API (v1) 3.1.0+2e23b97c3a
               fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+6a2618f53a
               fabric-command-api-v1: Fabric Command API (v1) 1.0.9+6a2618f53a
               fabric-commands-v0: Fabric Commands (v0) 0.2.0+6a2618f53a
               fabric-containers-v0: Fabric Containers (v0) 0.1.9+a03e98793a
               fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.0+e77439c73a
               fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d3a
               fabric-dimensions-v1: fabric-dimensions-v1 2.0.1+9a6c75813a
               fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.1+6a2618f53a
               fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+6a2618f53a
               fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.3+a4467d2a3a
               fabric-item-api-v1: Fabric Item API (v1) 1.2.0+6a2618f53a
               fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.1+6a2618f53a
               fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+730711c63a
               fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+6a2618f53a
               fabric-language-kotlin: Fabric Language Kotlin 1.4.0+build.1
               fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.0+ffb68a873a
               fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.1+6a2618f53a
               fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+6a2618f53a
               fabric-models-v0: Fabric Models (v0) 0.1.1+6a2618f53a
               fabric-networking-api-v1: Fabric Networking API (v1) 1.0.0+4358fbc63a
               fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.7+a03e98793a
               fabric-networking-v0: Fabric Networking (v0) 0.3.1+2a4333d33a
               fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.2+6a2618f53a
               fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.1+6a2618f53a
               fabric-particles-v1: fabric-particles-v1 0.2.2+6a2618f53a
               fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.3+be155ae23a
               fabric-renderer-api-v1: Fabric Renderer API (v1) 0.3.3+6a2618f53a
               fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.3+6a2618f53a
               fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.0+6a2618f53a
               fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.4+6a2618f53a
               fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.12+6a2618f53a
               fabric-rendering-v0: Fabric Rendering (v0) 1.1.1+6a2618f53a
               fabric-rendering-v1: Fabric Rendering (v1) 1.4.0+6a2618f53a
               fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.3.2+1944f3e03a
               fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+6a2618f53a
               fabric-structure-api-v1: Fabric Structure API (v1) 1.1.1+f1d8af063a
               fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.0+e77439c73a
               fabric-textures-v0: Fabric Textures (v0) 1.0.5+6a2618f53a
               fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.5+6a2618f53a
               fabricloader: Fabric Loader 0.10.8
               fallingleaves: Falling Leaves 1.2
               fancymenu: FancyMenu 1.6.0
               fastbench: FastBench 2.3
               fastfurnace: FastFurnace 2.3
               fatxporbs: Fat Experience Orbs 0.0.9
               fermion: Fermion 2.9.217
               fermion-modkeys: Fermion Modifier Keys 1.10.218
               fermion-orientation: Fermion Orientation 1.2.217
               fermion-varia: Fermion Varia 2.6.217
               fiber: fiber 0.23.0-2
               flonters: Flonters 1.1.1+1.16.2
               frex: FREX 4.2.183
               frex-events: FREX Events 1.1.11
               gbfabrictools: GBfabrictools 1.3.2+1.16.4
               geckolib3: Geckolib 3.0.3
               glassdoor: Glass Doors 1.5.0-1.16
               glowingbeds: Glowing Beds 1.2.1
               go-fish: Go Fish 1.1.0-beta-1.16.4
               goml: Get Off My Lawn 1.1.0
               gottaclimbfast: Gotta Climb Fast! 1.0
               gravestones: Gravestones v1.7
               gunpowder-base: Gunpowder 0.3.4+1.16.4
               gunpowder-sleepvote: Gunpowder SleepVote 1.0.3+gunpowder.0.3.1.mc.1.16.4
               horsestatsvanilla: Horse Stats Vanilla 4.1.4
               hplus: Hopper Plus 1.0.10
               iamverysmart: I Am Very Smart 2.1.0+mc1.16.1
               illuminations: Illuminations 1.3
               indrev: Industrial Revolution 1.7.11-BETA
               inmis: Inmis 1.3.1-1.16.2
               inventoryprofiles: Inventory Profiles 0.4.2
               inventorytabs: Inventory Tabs 0.3.0-1.16.4
               itemflexer: ItemFlexer 1.1.3
               jankson: Jankson 3.0.1+j1.2.0
               javax_inject_javax_inject: javax.inject 1
               javax_vecmath_vecmath: vecmath 1.5.2
               json-model-extensions: JSON Model Extensions 1.19.168
               killerbee: KillerBee 1.0
               kirin: Kirin UI 1.7-1.16.2
               konkrete: Konkrete 1.1.0
               kronhud: KronHUD 1.1.1
               krypton: Krypton 0.1.1
               kubejs: KubeJS 1.5.5
               lambdynlights: LambDynamicLights 1.3.2+1.16.4
               lapisreserve: Lapis Reserve 1.0.6
               launchersmod: Launchers 3.1.0
               lazydfu: LazyDFU 0.1.1
               leadvillagers: Lead Villagers 2.1.0+mc1.16.1
               libblockattributes: LibBlockAttributes 0.8.4
               libblockattributes_core: LibBlockAttributes (Core) 0.8.5
               libblockattributes_fluids: LibBlockAttributes (Fluids) 0.8.5
               libblockattributes_items: LibBlockAttributes (Items) 0.8.5
               libgui: LibGui 3.2.1+1.16.3
               library_of_exile: Library of Exile 1.0.6
               libstructure: LibStructure 1.5
               linkedstorage: Linked Storage 1.3.11-1.16
               lionutils: LionUtils 7.0.0+1.16
               lithium: Lithium 0.6.0
               lithium-api: Lithium API 0.6.0
               loadcatcher: Entrypoint Catcher 2.0.0
               loud_leads: Loud Leads 1.0.1+mc1.16.4
               maelstrom_library: Maelstrom Library 0.4.1
               magna: Magna 1.4.3-1.16.2
               mambience: MAmbience 2.1.0+1.16.2
               materialisation: Materialisation 3.0.3
               mcda: MC Dungeons Armors 0.9.3
               mcdw: MC Dungeons Weapons 2.6.0
               megane: megane 3.1.0+1.16.4
               megane-api: megane-api 3.1.0+1.16.4
               megane-impl: megane-impl 3.1.0+1.16.4
               megane-runtime: megane-runtime 3.1.0+1.16.4
               microconfig: Microconfig 1.0.3
               midnight-hats: Midnight Hats 2.0.1
               minecraft: Minecraft 1.16.4
               mixintrace: MixinTrace 1.0.0
               mmorpg: Age of Exile 1.7.8
               mo_glass: Mo Glass 1.5-MC1.16.4
               mobs_attempt_parkour: Mobs Attempt Parkour 0.2.1-1.16.3
               modern_industrialization: Modern Industrialization 0.3.7
               modid: Damage Tilt 1.16-fabric-0.1.0
               modmenu: Mod Menu 1.14.13+build.19
               monopoly: monopoly 1.2.1+minecraft-1.16.2
               monster-in-the-closet: Monsters in the Closet 1.0.0
               more_tooltips: MoreTooltips 1.0.1
               moreberries: More Berries 1.3.4
               moreblocksmod: More Blocks Mod 0.1.2.0
               muffle: Muffle 0.1.1
               mysql_mysql-connector-java: mysql-connector-java 5.1.46
               mythicmetals: Mythic Metals 0.9.5
               nbtcrafting: NBT Crafting 2.0.3+mc1.16.4
               notenoughcrashes: Not Enough Crashes 2.1.4+1.16.4
               nourish: Nourish 1.1.1
               numericping: NumericPing 1.0.2-fabric1.16.3
               okzoomer: Ok Zoomer 4.0.1+1.16.2
               org_apache_commons_commons-compress: commons-compress 1.19
               org_jetbrains_annotations: annotations 20.0.0
               org_jetbrains_exposed_exposed-core: exposed-core 0.27.1
               org_jetbrains_exposed_exposed-java-time: exposed-java-time 0.27.1
               org_jetbrains_exposed_exposed-jdbc: exposed-jdbc 0.27.1
               org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.4.0
               org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.4.0
               org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.4.0
               org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.4.0
               org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.3.9
               org_joml_joml: joml 1.10.0
               org_postgresql_postgresql: postgresql 42.2.2
               org_slf4j_slf4j-api: slf4j-api 1.8.0-beta1
               org_tukaani_xz: xz 1.8
               org_xerial_sqlite-jdbc: sqlite-jdbc 3.28.0
               org_yaml_snakeyaml: snakeyaml 1.26
               overloadedarmorbar: Overloaded Armor Bar 5.0.0-fabric
               patchouli: Patchouli 1.16.4-48-FABRIC
               piglib: Piglib 1.0.1
               pling: Pling 1.2.0
               polish: Polish 1.0.3
               polymorph: Polymorph 0.0.5-1.16.3
               presencefootsteps: Presence Footsteps r28+1.16.2
               pugh_lib: Pugh Lib 1.0.2
               randompatches: RandomPatches 1.16.1-0.0.0.5-fabric
               reach-entity-attributes: Reach Entity Attribute 1.0.1
               respawnablepets: Respawnable Pets 1.16.4-1
               retail: Retail 0.1.0
               rocks: This Rocks! 1.2.2
               roughlyenoughitems: Roughly Enough Items 5.8.10
               roughlyenoughitems-api: REI (API) 5.8.10
               roughlyenoughitems-default-plugin: REI (Default Plugin) 5.8.10
               roughlyenoughitems-runtime: REI (Runtime) 5.8.10
               roughlyenoughresources: Roughly Enough Resources 2.1.3
               rpg-hud: RPG-Hud 3.8.1
               rtree-3i-lite-fabric: rtree-3i-lite for Fabric 0.3.0
               sandwichable: Sandwichable 1.1.2
               shelf: Shelf 1.0.0
               shieldisruptor: Shield Disruptor 1.1.1
               simpledrawers: Simple Drawers 1.8.0-1.16.3
               slotlink: slotlink 2.2.0+1.16.4
               smithee: Smithee 0.1.5
               smokeextender: Smoke Extender 1.4.0
               smoothboot: Smooth Boot 1.16-1.3.1
               snowdrift: Snowdrift 1.2.1-1.16.4
               special-circumstances: Special Circumstances 1.8.217
               spinnery: Spinnery 3.1.13
               spruceui: SpruceUI 1.6.4
               starlight: Starlight 0.0.3
               static-content: Static Content 1.0.1-1.16.2
               staticdata: Static Data 1.1.2
               strongandfairanvils: Strong and Fair Anvils 0.3.0+1.16
               tacobell: Taco Bell 1.1.1
               tcrafting: TCrafting 1.2.1-1.16.4
               team_reborn_energy: Energy 0.1.1
               tellme: TellMe 0.9.1-beta.1
               terraform-biome-builder-api-v1: Terraform Biome Builder API (v1) 1.0.0+build.1
               terraform-config-api-v1: Terraform Config API (v1) 1.0.0+build.1
               terraform-dirt-api-v1: Terraform Dirt API (v1) 1.0.0+build.1
               terraform-overworld-biome-extensions-api-v1: Terraform Overworld Biome Extensions API (v1) 1.0.0+build.1
               terraform-shapes-api-v1: Terraform Shapes API (v1) 1.0.0+build.4
               terraform-surfaces-api-v1: Terraform Surfaces API (v1) 1.0.0+build.1
               terraform-tree-api-v1: Terraform Tree API (v1) 1.0.1+build.3
               terraform-wood-api-v1: Terraform Wood API (v1) 1.0.0+build.1
               terrestria: Terrestria 2.1.5+build.28
               textile_backup: Textile Backup 2.0.1-1.16.3
               the_loved_ones: The Loved Ones 1.1.0+1.16.4
               tinytweaks: TinyTweaks 1.1.2
               tis3d: TIS-3D 1.6.1
               tooltipfix: ToolTip Fix 1.0.2-1.16
               torohealth: ToroHealth Damage Indicators 1.16.4-fabric-4
               trinkets: Trinkets 2.6.7
               unforgivingvoid: Unforgiving Void 2.0.0
               universalcomponents: Universal Components 0.7.3+1.16.2
               veggie_way: The Veggie Way 1.0.35
               villagernames: Villager Names 1.6.0
               visualoverhaul: VisualOverhaul 1.0.0
               voxelmap: VoxelMap 1.10.14
               waila: Hwyla 1.9.22
               wild_explorer: Wild Explorer 1.3
               wolveswitharmor: Wolves With Armor 1.5.2-1.16.2
               woodenhoppers: Wooden Hoppers 1.0.0
               woods_and_mires: Woods and Mires 1.0.4+1.16.3
               wrenchable: Wrenchable 1.0.0+1.16.1
               ynet: YNet 0.2.0
               yosbr: YOSBR 0.1.1
               yungsapi: YUNG's API 1.16.4-Fabric-1
               zoob: Monster Of The Ocean Depths 2.5.0
```

**Comment by ExcessiveAmountsOfZombies** [comment]
is this a public modpack? Does the game ever get to the main menu or is it just stuck loading forever? This resource pack probably won't help if it never gets to the main menu but if it does mind trying it out and seeing if it stops the errors?
[testpack.zip](https://github.com/ExcessiveAmountsOfZombies/Croptopia/files/5929920/testpack.zip)

**Comment by ExcessiveAmountsOfZombies** [comment]
if you want a jar with the changes because it doesn't load, you'll have to unzip it, only way i could upload it
[croptopia.zip](https://github.com/ExcessiveAmountsOfZombies/Croptopia/files/5929959/croptopia.zip)

**Comment by MechanosG** [comment]
The pack will eventually be public and is currently in testing.  But yes, the game gets to the main menu and singleplayer/multiplayer work fine.  I'll test that resource pack.

**Comment by MechanosG** [comment]
The resource pack seems to have stopped the log spam.  :)

**Comment by MechanosG** [comment]
So unfortunately I have news.  I updated my modpack to 1.16.5, and that resource pack is no longer stopping the log spam.  I don't know why updating from 1.16.4 to 1.16.5 would change things, but it did for some reason.  Starting at line 760:
```
[13:57:35] [Worker-Main-2/WARN]: Unable to resolve texture due to reference chain crop->crop in croptopia:block/croptopia_leaves
```
That gets spammed 'till line 375159.  The game menu does load though.

**Comment by ExcessiveAmountsOfZombies** [comment]
this might be fixed in 1.3.0 now. originally I thought it was ARRP causing it but AOF has it too and no warnings there.

**Comment by GordonHollow** [comment]
Has the same console spam on latest 1.18.2 release.
Play modpack ATM7 + Optifine
